sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/block/authority/AuthorityRuleChecker.java

```diff
@@ -29,39 +29,39 @@
 
     static boolean passCheck(AuthorityRule rule, Context context) {
         String requester = context.getOrigin();
+        boolean contain = false;
 
-        // Empty origin or empty limitApp will pass.
-        if (StringUtil.isEmpty(requester) || StringUtil.isEmpty(rule.getLimitApp())) {
-            return true;
+        {
+            // Empty origin or empty limitApp will pass.
+            if (StringUtil.isEmpty(requester) || StringUtil.isEmpty(rule.getLimitApp())) {
+                return true;
+            }
         }
```

Comment:
Why are there extra { } here

Reply:
This piece of logic is distinguished by a code block, which makes it feel clearer

Reply:
Thank you for your reply.But I didn't see this code style elsewhere in this project. I think we should unify the code style.

Reply:
What you said is quite reasonable. I have removed this part of the logic